pydantic_ai_slim/pydantic_ai/_output.py

```diff
@@ -835,13 +929,22 @@ async def process(
         self,
         data: str,
         run_context: RunContext[AgentDepsT],
+        trace_context: TraceContext,
         allow_partial: bool = False,
         wrap_validation_errors: bool = True,
     ) -> OutputDataT:
         args = {self._str_argument_name: data}
 
         try:
-            output = await self._function_schema.call(args, run_context)
+            # Wraps the output function call in an OpenTelemetry span if trace_context is provided.
+            # Note: PlainTextOutputProcessor is used for text responses (not tool calls),
+            # so we don't have tool call attributes like gen_ai.tool.name or gen_ai.tool.call.id
+            function_name = getattr(self._function_schema.function, '__name__', 'text_output_function')
+            call = _messages.ToolCallPart(tool_name=function_name, args=args)
```

Comment:
these should not be inside the `try`, same for the other execute_function_with_span